railties/test/generators/actions_test.rb

```diff
@@ -71,10 +71,17 @@ def test_gem_should_put_gem_dependency_in_gemfile
 
   def test_gem_with_version_should_include_version_in_gemfile
     run_generator
-
-    action :gem, "rspec", ">=2.0.0.a5"
-
-    assert_file "Gemfile", /gem 'rspec', '>=2.0.0.a5'/
+    action :gem, "rspec", ">= 2.0.0.a5"
+    action :gem, "RedCloth", ">= 4.1.0", "< 4.2.0"
+    action :gem, "nokogiri", version: ">= 1.4.2"
+    action :gem, "faker", version: [">= 0.1.0", "< 0.3.0"]
+
+    assert_file "Gemfile" do |content|
+      assert_match /gem 'rspec', '>= 2\.0\.0\.a5'/, content
```

Comment:
This will cause Ruby warnings as shown below.

```
test/generators/actions_test.rb:81: warning: ambiguous first argument; put parentheses or a space even after `/' operator
````

You need to change like following.

```ruby 
assert_match(/gem 'rspec', '>= 2\.0\.0\.a5'/, content)
```